deleteAllWords.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import json
+from config import Config
+from ibm_watson import TextToSpeechV1
+from ibm_cloud_sdk_core.authenticators import IAMAuthenticator
+
+# python deleteAllWords.py config.ini
+
+config_file = "config.ini"
+if len(sys.argv) > 1:
+    config_file = sys.argv[1]
+else:
+    print("Using default config filename: config.ini.")
+
+config      = Config(config_file)
+apikey = config.getValue("TextToSpeech", "apikey")
+url = config.getValue("TextToSpeech", "service_url")
+customization_id = config.getValue("TextToSpeech", "customization_id")
+
+authenticator = IAMAuthenticator(apikey)
+text_to_speech = TextToSpeechV1(
+    authenticator=authenticator
+)
+
+text_to_speech.set_service_url(url)
+
+# get list of words
+word_list = []
+words_raw = text_to_speech.list_words(customization_id).get_result()
+for element in words_raw['words']:
+    word_list.append(element['word'])
+print("existing word list: ", word_list)
+
+# loop through deleting the words
+for word in word_list:
+    text_to_speech.delete_word(customization_id, word)
+    print("deleted word: ", word)
```

Comment:
Let's chat, either we always print progress or never, in these tools.

Reply:
I prefer to print progress. I believe the "main" scripts print progress as well. This scripts specifically should be moved to "models.py" as mentioned in another comment

Reply:
Moved print to stderr so that stdout can be safely redirected to output file.